### ISSUE TITLE
[REQUEST] Item restrict to checkcommandaccess

### ISSUE THREAD

**Issue by ByDexterTR** [nuclearsilo583/zephyrus-store-preview-new-syntax#96]
It would be better if a command restriction such as **"checkcommandaccess" "sm_ban"** was added to an item we added to the market instead of a flag restriction.

**Comment by azalty** [comment]
Good idea, it would allow setting custom overrides too, and allow admin groups instead of only flags
